ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-jaxrs/main/module.xml

```diff
@@ -47,6 +47,6 @@
         <module name="org.jboss.resteasy.resteasy-validator-provider" optional="true" services="export" export="true"/>
         <module name="org.jboss.logging"/>
         <module name="org.reactivestreams"/>
-        <module name="org.eclipse.microprofile.config.api"/>
+        <module name="org.eclipse.microprofile.config.api" optional="true"/>
```

Comment:
@ronsigal Could you please confirm that it is fine with resteasy if this module is not present?

Reply:
Hi @jmesnil,

Sorry for the late reply. I have believed that it would be fine, but I took another look at the code and it looks ok. I was going to run the WildFly testsuite to convince myself even further, but then I noticed that resteasy-jaxrs/main/module.xml in RESTEasy is

```
    <dependencies>
        ...
        <module name="org.eclipse.microprofile.config.api" export="true" optional="true"/>
    </dependencies>
```
so, there's the answer. It's fine.

Reply:
thanks!